openapi/openai.chat/Module.md

```diff
@@ -0,0 +1,51 @@
+
+## Overview
+
+This is a generated connector for the [OpenAI Chat API](https://beta.openai.com/docs/api-reference/chat) OpenAPI Specification. OpenAI is an American artificial intelligence research laboratory consisting of a non-profit corporation and a for-profit subsidiary. OpenAI conducts AI research with the declared intention of promoting and developing friendly AI. The OpenAI Chat API provides a way to access the state-of-the-art ChatGPT model (GPT-3.5) developed by OpenAI for a variety of tasks.
```

Comment:
Do we need to mention that the connector supports only GPT-3.5 varients? Can't we specify other models (e.g GPT 4) in the completion request and retrieve responses in the same way?

Reply:
As of now, only GPT 3.5 Turbo model is supported for chat completions. Updated the content.